frontend/file_form.js

```diff
@@ -95,12 +95,19 @@ class RenderUploadFile {
     return this.container.querySelector(`.dff-file-id-${index}`);
   }
 
-  setSuccess(index) {
+  setSuccess(index, size) {
     const { translations } = this;
 
     const el = this.findFileDiv(index);
     el.classList.add("dff-upload-success");
 
+    const fileSizeInfo = document.createElement("span");
+    fileSizeInfo.innerHTML = size;
+    fileSizeInfo.className = "dff-filesize";
+    fileSizeInfo.setAttribute("data-index", index);
```

Comment:
I think it's not necessary to set the `data-index`. It is added for the delete link because it is used in the click handler.